src/main/java/bio/terra/cli/command/resource/Resolve.java

```diff
@@ -60,6 +61,17 @@ protected void execute() {
       default:
         cloudId = resource.resolve();
     }
-    formatOption.printReturnValue(cloudId);
+    JSONObject object = new JSONObject();
```

Comment:
alternatively, I can create a JSON serializable resolve class for each individual resources. But i'd like the  JSON object's key to be the resource name.

The datasource resolve will need a jsonarray of resource _name: resource_cloud_id to be easily dereferenced.